flexget/utils/search.py

```diff
@@ -11,7 +11,16 @@ def clean_symbols(text):
     result = text
     if isinstance(result, unicode):
         result = normalize('NFKD', result)
-    return re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    result = re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    
+    """Special cases"""
```

Comment:
Regular comments should just begin with a `#`. And I'm still not quite convinced this is the best approach in what is meant to be a general utility. Might not some (less smart) search engines have a problem giving results like `marvel's` when the input was `marvels`?